src/main/java/com/ericsson/ei/erqueryservice/ERQueryService.java

```diff
@@ -53,6 +53,10 @@
     @Value("${event.repository.url}")
     private String eventRepositoryUrl;
 
+    @Getter
+    @Value("${event.repository.shallow}")
+    private String shallowParameter;
```

Comment:
cant you reasd this as boolean

Reply:
This shallowParameter value is required as string while creating http request to ER . Thats why kept it as string.